br/pkg/lightning/backend/kv/sql2kv.go

```diff
@@ -433,7 +434,8 @@ func (kvcodec *tableKVEncoder) Encode(
 	}
 	kvPairs := kvcodec.se.takeKvPairs()
 	for i := 0; i < len(kvPairs.pairs); i++ {
-		kvPairs.pairs[i].RowID = rowID
+		var encoded [8]byte
+		kvPairs.pairs[i].RowID = codec.EncodeInt(encoded[:0], rowID)
```

Comment:
Is there any memcomparable varint-encoded format of the int64 rowID?  Since we will add extra two bytes in `keyAdapter`, it would be better if we can save some bytes here.

Reply:
Seems `codec.EncodeComparableVarint` is okay for this.

Reply:
Updated